pages/blueprints/index.js

```diff
@@ -95,8 +95,11 @@ class BlueprintsPage extends React.Component {
           manageSources={manageSources}
         />
         {(blueprintsLoading === true && <Loading />) ||
+          (blueprintsError === undefined && (
+            <EmptyStateInactive fetchingBlueprints={this.props.fetchingBlueprints} />
+          )) ||
           (blueprintsError !== null &&
```

Comment:
Just asking, but if the default state is changed from `null` to `undefined` in `store.js`, then should this line also be changed from `null` to `undefined`?